Makefile

```diff
@@ -140,8 +140,11 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | kubectl apply --server-side -f -
 	for w in statefulset/trivy deployment/image-scanner-controller-manager; do \
-  		kubectl rollout status $$w -n $(K8S_NAMESPACE) --timeout=2m \
-  		|| (kubectl logs -n $(K8S_NAMESPACE) $$w --tail -1; kubectl get events -n $(K8S_NAMESPACE); false); \
+		if ! kubectl rollout status $$w -n $(K8S_NAMESPACE) --timeout=2m; then \
+			kubectl logs -n $(K8S_NAMESPACE) $$w --tail -1; \
+			kubectl get events -n $(K8S_NAMESPACE); \
```

Comment:
Maybe we should print the events first? What happens if the workload is not scheduled? Will the `kubectl logs` command fail?

Reply:
I think we should change the order, since `kubectl logs` seems to fail when the workload is not scheduled, and `kubectl get events` is never run. Ref: https://github.com/statnett/image-scanner-operator/actions/runs/3884117789/jobs/6626514357#step:8:125

Reply:
done